src/components/link/link.style.js

```diff
@@ -78,15 +78,21 @@ const LinkStyle = styled.span`
   `}
 `;
 
-LinkStyle.defaultProps = {
+StyledLink.defaultProps = {
   theme: baseTheme,
   disabled: false,
   hasContent: true,
 };
 
-LinkStyle.propTypes = {
+StyledLink.propTypes = {
   disabled: PropTypes.bool,
   hasContent: PropTypes.bool,
 };
 
-export default LinkStyle;
+const StyledContent = styled.span``;
```

Comment:
This can be removed, there's no need for an empty styled component, can just use a `span` instead

Reply:
In general yes, but because we remove className this is the clearest way to have a link content selector that we use in pod.style.js

Reply:
Ok fair enough, I see why you've done that now 👍🏻